lib/shopify_transporter.rb

```diff
@@ -235,7 +235,12 @@ def valid_file?(path)
 
   def build_classes_based_on_config
     @record_class = Object.const_get("ShopifyTransporter::Shopify::#{@object_type.capitalize}")
-    @record_builder = ShopifyTransporter::RecordBuilder.new(
+    record_builder_class = if @object_type.capitalize == 'Product'
```

Comment:
I think this is the best option in terms of complexity versus value:

>Perhaps there needs to be a record builder for each object type (where they all just call super except product) 

since you already have the `@record_class` here.

I agree that this isn't optimal but I'd be ok with it going in for now since we are on a time limit and this isn't so complex that it _requires_ an abstraction.